src/AjaxDynamicDataTable.jsx

```diff
@@ -113,12 +113,18 @@ class AjaxDynamicDataTable extends Component {
 
 }
 
+let axios;
+
+if (typeof window !== 'undefined') {
+    axios = window.axios
+}
+
 AjaxDynamicDataTable.defaultProps = {
     onLoad: () => null,
     params: {},
     defaultOrderByField: null,
     defaultOrderByDirection: null,
-    axios: window.axios || require('axios'),
+    axios: axios || require('axios'),
```

Comment:
```suggestion
    axios: (window && window.axios) ? window.axios : require('axios'),
```

Perhaps just do the above, rather than adding to global scope?

Reply:
@DivineOmega This actually still causes an exception.

Reply:
I've updated your suggestion to check the type of the variable, as this is safe compared to checking the value of a variable.